### DISCUSSION TITLE
gss-ntlmssp dependencies and installation instructions?

### DISCUSSION THREAD

**Discussion by Ichabaud** [gssapi/gss-ntlmssp#81] [Q&A] [answered]
Hi Team,

First, thank you for your hard work contributing to this code.

I work for a small company that uses an Alma-Linux 8.4 installation running Microsoft Powershell for a variety of purposes.

We want to update our 0.7.0-6 gss-ntlmssp to the newer 1.0 version.  It seems that the 1.0 (and 1.1) version of gss-ntlmssp have some dependencies that we are unable to work our way around.

Specifically, we see this error during manual installation:
     configure error: "wbclient headers not found"

It seems we need the Samba Winbind client which we have installed.  What steps or dependencies are we missing to complete the installation?

Appreciate your time and patience,

Ichabaud

**Comment by simo5** [comment, marked as answer]
./configure --with-wbclient=no
